pkgs/by-name/ap/appium-inspector/package.nix

```diff
@@ -0,0 +1,61 @@
+{ fetchFromGitHub
+, buildNpmPackage
+, makeWrapper
+, electron_13
+, makeDesktopItem
+, stdenv
+, lib
+}:
+let
+  electron = electron_13;
```

Comment:
This is very old, can't we use something newer?

Reply:
I also mentioned this to the upstream maintainers, it doesn't work on anything newer unfortunately 
see https://github.com/appium/appium-inspector/issues/1321#issuecomment-1926911425